Modules/Quotas/src/Quotas.Infrastructure.Database.SqlServer/Migrations/QuotasDbContextModelSnapshot.cs

```diff
@@ -27,6 +27,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<string>("Id")
                         .HasColumnType("nvarchar(450)");
 
+                    b.Property<int>("CipherSize")
+                        .HasColumnType("int");

```

Comment:
You have to recreate the SQL Server snapshot after the change of the column type. It's still `int` here.